docs/Tutorials/Privacy/Private-Transaction-Performance-Best-Practice.md

```diff
@@ -0,0 +1,73 @@
+# Private transaction performance best practices
+
+This document collects deployment and usage hints to help achieving high performance for private transaction.
+If transaction throughput or latency is not meeting your expectations, please consider the following before raising the issue.
+
+## General performance
+
+Private transaction use the same facilities as public ones.
+General Besu performance tunings applies.
+Specific approaches are out of scope of this document.
+However the following item is of such outstanding impact to performance that it is  mentioned here:
+
+### Use fast, local, solid state storage
+
+Running EVM transactions creates a lot of random reads that are executed sequentially.
+The Besu data folder should be located on the fastest possible storage media.
+Prefer NVMe attached SLC flash or Intel Optane.
+Avoid network attached SSDs or cloud storage with limited IOPs.
+Do not use spinning disks under any circumstances.
+
+## Private transaction performance
+
+### Use concurrent submission
+
+When submitting a private transaction using [web3js-quorum](https://github.com/ConsenSys/web3js-quorum), the submit call will only return once the PMT transaction has been included in a block.
+This limits the throughput to at most one private transaction per block when submitting from a single thread.
+To increase throughput, use web3js-quorum from multiple concurrent threads or processes.
+
+### Colocate Besu and Tessera
+
+Besu has to talk to its local Tessera node frequently while handling a block.
+While it is recommended to not run them on the same node, one should minimize the latency between Besu and Tessera to improve block processing times.
```

Comment:
While we do not recommend running them on the same node, minimizing the latency between Besu and Tessera will improve block processing times.